### ISSUE TITLE
Coinbase & Coinbase Exchange 

### ISSUE THREAD

**Issue by lawrencegs** [mobnetic/BitcoinChecker#229]
Coinbase & Coinbase Exchange used to be two different exchanges (they have different price). Since 1.6.3 I can only see Coinbase. What happen to Coinbase Exchange?

**Comment by tauren** [comment]
I agree. It isn't clear if Coinbase in the app is the main Coinbase buy/sell platform, or if it is the Coinbase Exchange (which is now called GDAX). I'm guessing it is the main Coinbase platform since it shows bid/ask prices instead of high/low prices. I'm surprised that GDAX isn't yet an option.

**Comment by 0xBADCODE** [comment]
Can this be correcting in the app, I assume the coinbase option is now the GDAX exchange? Needs to be renamed.
